vault/raft.go

```diff
@@ -406,14 +422,27 @@ func (c *Core) startPeriodicRaftTLSRotate(ctx context.Context) error {
 	return nil
 }
 
-func (c *Core) createRaftTLSKeyring(ctx context.Context) error {
-	if _, ok := c.underlyingPhysical.(*raft.RaftBackend); !ok {
-		return nil
+// RaftCreateTLSKeyring creates the initial TLS key and the TLS Keyring for raft
+// use. If a keyring entry is already present in storage, it will return an
+// error.
+func (c *Core) RaftCreateTLSKeyring(ctx context.Context) (*raft.TLSKeyring, error) {
```

Comment:
Any reason why you made this exported?

Reply:
Nope, I had to use this outside of the package at some point before deciding to revert. I'll revert this change.